src/azure-cli/azure/cli/command_modules/batch/commands.py

```diff
@@ -141,7 +140,6 @@ def get_data_factory(name):
         g.batch_command('delete', 'remove_nodes')
 
     with self.command_group('batch job', get_data_type('job'), client_factory=get_data_factory('job')) as g:
```

Comment:
We can also add the deprecating info to certificate related commands, docs: https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments